source/security/client-side-field-level-encryption-guide.txt

```diff
@@ -262,6 +262,24 @@ Additional Dependencies
              <https://pypi.org/project/pymongocrypt/>`_
            - Python wrapper for the ``libmongocrypt`` encryption library.
 
+   .. tab::
+      :tabid: csharp
+
+      .. list-table::
+         :header-rows: 1
+
+         * - Dependency Name
+           - Description
+
+         * - x64 Support
+           - x64 support is required for CSFLE
+
+
+         * - `libmongocrypt
+             <https://github.com/mongodb/libmongocrypt#installing-libmongocrypt-on-windows>`__
+           - A Windows build of the `libmongocrypt`
```

Comment:
Would it be useful to specify DLL here?
Also nitpick: according to the style guide, list items should start with caps and end with a period.
I'm certain there was another acceptable version in which you start with lowercase and end without period, but it appears to have been removed from our official style guide.